CDP4OfficeInfrastructure/OfficeDal/CrossviewWorkbookData.cs

```diff
@@ -43,96 +46,96 @@ namespace CDP4OfficeInfrastructure.OfficeDal
+            set => this.selectedElementDefinitions = value.Value;
         }
 
         /// <summary>
         /// Gets or sets the <see cref="IEnumerable{ParameterType}"/> data of the current <see cref="Workbook"/>
         /// </summary>
         [XmlElement(typeof(XmlCDataSection))]
-        public XmlCDataSection ParameterTypeData
+        public XmlCDataSection SelectedParameterTypes
         {
             get
             {
                 var doc = new XmlDocument();
-                return doc.CreateCDataSection(this.parameterTypeData);
+                return doc.CreateCDataSection(this.selectedParameterTypes);
             }
 
-            set => this.parameterTypeData = value.Value;
+            set => this.selectedParameterTypes = value.Value;
         }
 
         /// <summary>
         /// Gets or set the modefied cell names of the current <see cref="Workbook"/>
         /// </summary>
         [XmlElement(typeof(XmlCDataSection))]
-        public XmlCDataSection CellNamesData
+        public XmlCDataSection CellNames
         {
             get
             {
                 var doc = new XmlDocument();
-                return doc.CreateCDataSection(this.cellNamesData);
+                return doc.CreateCDataSection(this.cellNames);
             }
 
-            set => this.cellNamesData = value.Value;
+            set => this.cellNames = value.Value;
         }
 
         /// <summary>
         /// Gets or set the modefied cell values of the current <see cref="Workbook"/>
         /// </summary>
         [XmlElement(typeof(XmlCDataSection))]
-        public XmlCDataSection CellValuesData
+        public XmlCDataSection CellValues
         {
             get
             {
                 var doc = new XmlDocument();
-                return doc.CreateCDataSection(this.cellValuesData);
+                return doc.CreateCDataSection(this.cellValues);
             }
 
-            set => this.cellValuesData = value.Value;
+            set => this.cellValues = value.Value;
         }
 
         /// <summary>
         /// Gets the <see cref="IEnumerable{ElementDefinition}"/> instances.
         /// </summary>
         [XmlIgnore]
-        public IEnumerable<Thing> ElementDefinitionList => this.Serializer.Deserialize(this.GenerateStreamFromString(this.ElementDefinitionData.Value));
+        public IEnumerable<Thing> ManuallySavedElementDefinitionValues => this.Serializer.Deserialize(this.GenerateStreamFromString(this.SelectedElementDefinitions.Value));
 
         /// <summary>
         /// Gets the <see cref="IEnumerable{ParameterType}"/> instances.
         /// </summary>
         [XmlIgnore]
-        public IEnumerable<Thing> ParameterTypeList => this.Serializer.Deserialize(this.GenerateStreamFromString(this.ParameterTypeData.Value));
+        public IEnumerable<Thing> ManuallySavedParameterValues => this.Serializer.Deserialize(this.GenerateStreamFromString(this.SelectedParameterTypes.Value));
```

Comment:
Rename to `SavedParameterTypes`: they are not "manually added values" as in the case of the cells.